sympy/concrete/summations.py

```diff
@@ -512,6 +512,21 @@ def is_convergent(self):
         except NotImplementedError:
             pass
 
+        ### ---------- Raabe's test -------------- ###
+        if lim_ratio == 1:  # ratio test inconclusive
+            test_val = sym*(sequence_term/
+                         sequence_term.subs(sym, sym + 1) - 1)
+            try:
+                lim_val = limit_seq(test_val, sym)
+                if lim_val is not None and lim_val.is_number:
+                    if lim_val > 1:
```

Comment:
I think we can check `(lim_val - 1).is_positive` here, to allow it to potentially work for symbols with the proper assumptions.

Reply:
OK will change the line if lim_value > 1: